.gitignore

```diff
@@ -33,3 +33,7 @@ $RECYCLE.BIN/
 
 # Mac
 .DS_Store
+
+# Stuff
+reference114
```

Comment:
can you please not add random stuff to the gitignore.

Reply:
sorry i need that one

Reply:
We can't be adding random stuff that exists on your system to the github gitignore

Reply:
thats the folder i used for the 1.14 fabric api source code
its fine cmon

Reply:
No its not, just move your source code to a different directory.

Reply:
irritat